app/commands.py

```diff
@@ -765,3 +765,31 @@ def create_user_jwt(token):
     service_id = token[-73:-37]
     api_key = token[-36:]
     print(create_jwt_token(api_key, service_id))
+
+
+def update_template(id, name, template_type, content, subject):
+    update = """
+        UPDATE {} SET name = '{}', template_type = '{}', content = '{}', subject = '{}'
+        WHERE id = '{}'
+    """
+
+    for table_name in 'templates', 'templates_history':
+        db.session.execute(
+            update.format(
```

Comment:
[suggestion] is there a better way to run this command in alchemy than string.format? I'm a little surprised bandit didn't flag this.

Even though the data is coming from trusted sources, I'd like a little more protection from ourselves.

Reply:
This should be improved now, thanks for the nudge.